### PR TITLE
US 1583733 Add missing language IDs - 12

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "output" for program output examples.
- Used "csharp" for ADO.NET connection string examples.
- Used "text" for examples of the relation between DataSet components.

Contributes to #2192

### CODE REVIEW

File: docs/framework/data/adonet/connection-string-syntax.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ When you connect to Azure SQL Database or to Azure SQL Data Warehouse and provid
 ### Connect to a named instance of SQL Server
 To connect to a named instance of SQL Server, use the *server name\instance name* syntax.  
   
-```  
+```csharp  
 Data Source=MySqlServer\MSSQL1;"  
 ```
```
Comment (Youssef1313) [inline docs/framework/data/adonet/connection-string-syntax.md]: That's not related to the language identifier, I think there is a missing double quote.

Comment (v-thpra) [reply docs/framework/data/adonet/connection-string-syntax.md]: Good catch. I made the fix.

**Comment by v-thpra** [comment]
@mairaw Conflicts and changes have been resolved. This should be ready to merge. Thanks!

**Review by BillWagner — APPROVED** [review]
This LGTM @v-thpra 

I'll :shipit: now.

/

**Pull request merged**